app/gui/Cargo.toml

```diff
@@ -47,7 +47,10 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0" }
 sha3 = { version = "0.8.2" }
 uuid = { version = "0.8", features = ["serde", "v4", "wasm-bindgen"] }
-wasm-bindgen = { version = "=0.2.58" }
+# wasm-bindgen version 0.2.78 is causing issues with clippy.
+# See for more information. https://github.com/rustwasm/wasm-bindgen/issues/2774
+# Should be removed once 0.2.80 is available.
+wasm-bindgen = { version = "=0.2.78" }
```

Comment:
I'm a bit confused. We restrict the `wasm-bidgen` version here, but comments says that _this_ version is causing issues. Nevertheless, the `Cargo.lock` has the old version locked only. How does it work?

Also, the new version of `wasm-bindgen` may cause the problems in our build scripts, as we patch the output WASM (See `js_workaround_patcher` in run.js file).

Reply:
Good point! This needs updating.

Reply:
This also fails CI. For some reason, it did not appear locally.

Reply:
Actually, I have looked at the wrong version of `Cargo.lock`. On this branch, the `wasm-bindgen` is in the new version everywhere, so I'm no longer confused.